### PR TITLE
Add more browser flags

### PR DESCRIPTION
**Pull request by monica-ch**
Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/webview-features-flags?branch=pr-en-us-3099#available-webview2-browser-flags

This PR adds more browser flags to the documentation.

AB#49406538

### CODE REVIEW

File: microsoft-edge/webview2/concepts/webview-features-flags.md [hunk 1]
```diff
@@ -77,6 +78,7 @@ The following are some of the flags we've seen used.
 | `disk-cache-size` | Forces the maximum disk space to be used by the disk cache, in bytes. |
 | `do-not-de-elevate` | Prevents de-elevation of the WebView2 on launch.  Used after de-elevating, to prevent infinite loops. |
 | `edge-webview-debugging-script` | Runs `PostWebMessage` script asynchronously. |
+| `edge-webview-enable-mojo-ipcz` | Enables MojoIpcz (mojo-ipcz) for apps that use WebView2.  See [todo]. |
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/webview-features-flags.md:81]: what url for info about MojoIpcz?  @monica-ch

File: microsoft-edge/webview2/concepts/webview-features-flags.md [hunk 2]
```diff
@@ -109,9 +111,12 @@ The following are some of the flags we've seen used.
 | `msOverlayScrollbarWinStyle` | Whether the users can change between overlay and non-overlay modes for Fluent scrollbars. |
 | `msPdfEnableAsPreview` | This features enables the PDF viewer to launch with a minimal toolbar and in read-only preview mode. |
 | `msSmartScreenProtection` | If enabled, SmartScreen protection will be available. |
+| `msUseSpellCheckCorrectionsCard` | If enabled, a new corrections card UI is shown when the user clicks on the error range of a misspelling or grammar error. |
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/webview-features-flags.md:114]:
```suggestion
| `msUseSpellCheckCorrectionsCard` | If enabled, a new corrections card UI is shown when the user clicks in a range of code that includes a misspelling or grammar error. |
```
@monica-ch - review this Suggested Change.  trying to clarify text in this cell.

File: microsoft-edge/webview2/concepts/webview-features-flags.md [hunk 3]
```diff
@@ -120,12 +125,13 @@ The following are some of the flags we've seen used.
...
 | `proxy-server` | A proxy server that overrides system settings.  This flag only affects HTTP and HTTPS requests. |
-| `remote-allow-origins` | Enables web socket connections from the specified origins only.  The '*' wildcard allows any origin. |
+| `remote-allow-origins` | Enables web socket connections from the specified origins only.  The `*` wildcard allows any origin. |
 | `remote-debugging-port` | Enables remote debugging over HTTP on the specified port. |
 | `RendererAppContainer` | Enables Renderer `AppContainer`. |
 | `Restart` | Indicates that Microsoft Edge WebView2 browser process was restarted (such as after a flag change).  Use this flag to ignore the launch when recording the `Launch.Mode2` metric. |
 | `sdsm-state` | The "Super Duper Secure Mode" state.  Valid values: `off`, `basic`, `balanced`, `strict`. |
 | `SharedArrayBuffer` | If enabled, indicates that a `SharedArrayBuffer` thread is present. |
+| `SpareRendererForSitePerProcess` | Controls whether Edge Webview spare renderer process tries to always have a warm spare renderer process around for the most recently requested BrowserContext.  This feature is only consulted in site-per-process mode. |
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/webview-features-flags.md:134]:
```suggestion
| `SpareRendererForSitePerProcess` | Controls whether the Edge Webview spare renderer process tries to always have a warm spare renderer process running for the most recently requested `BrowserContext`.  This feature is only consulted in site-per-process mode. |
```
tech review these changes @monica-ch

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Writer/Editor review: requested changes, eg need link for MojoIpcs.

**Review by nishitha-burman — APPROVED** [review]

File: microsoft-edge/webview2/concepts/webview-features-flags.md [hunk 4]
```diff
@@ -106,8 +106,11 @@ The following are some of the flags we've seen used.
 | `msAbydosGestureSupport` | Allows users to use gestures (such as the scratchout gesture) to delete text by using a pen.  Valid only if the `msAbydos` flag is enabled. |
 | `msAbydosHandwritingAttr` | Whether the "handwriting-to-text" experience is enabled for input elements at the DOM level.  Valid only if the `msAbydos` flag is enabled. |
 | `msEdgeFluentOverlayScrollbar` | Force-enables Fluent Overlay scrollbars, overriding whatever value `kFluentScrollbar` may hold at the moment. |
+| `msEdgeDevToolsWdpRemoteDebugging` | Enables remote debugging. |
 | `msEnhancedTextContrast` | Improves text contrast enhancement and gamma correction to match the quality and clarity of other native Windows applications.  When this flag is used, font rendering respects user ClearType Tuner settings when applying text contrast enhancement and gamma correction. |
 | `msEnhancedTrackingPreventionEnabled` | Enables native privacy protection features, such as blocking cookies and web requests that reside in domains that are known to be tracking domains. |
+| `msFloatyMode` | Use this to disable Floaty feature, as WebView doesn't support to browser retention experiments. |
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/webview-features-flags.md:112]: link to Floaty feature info, or define Floaty feature.  How is Floaty related to browser retention?  Define/clarify "browser retention" (add hint-words).  Cell needs to give text that's meaningful to our audience.
@monica-ch

File: microsoft-edge/webview2/concepts/webview-features-flags.md [hunk 5]
```diff
@@ -106,8 +106,11 @@ The following are some of the flags we've seen used.
...
+| `msFloatyShouldHonorIndiaHoldout` | Use this to disable Floaty India holdout feature that enables if the user is part of control of the India holdout group, as WebView doesn't support browser retention experiments. |
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/webview-features-flags.md:118]: "the India holdout group" - is this flag Microsoft internal-only?  @monica-ch , cc @captainbrosset

File: microsoft-edge/webview2/concepts/webview-features-flags.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/webview-features-flags.md]: removed todo; out of scope

File: microsoft-edge/webview2/concepts/webview-features-flags.md (hunk 4 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/concepts/webview-features-flags.md]: defining is probably out of scope

File: microsoft-edge/webview2/concepts/webview-features-flags.md [hunk 6]
```diff
@@ -103,36 +105,51 @@ The following are some of the flags we've seen used.
 | `msAbydos` | Enables the "handwriting-to-text" experience. |
 | `msAbydosGestureSupport` | Allows users to use gestures (such as the scratchout gesture) to delete text by using a pen.  Valid only if the `msAbydos` flag is enabled. |
 | `msAbydosHandwritingAttr` | Whether the "handwriting-to-text" experience is enabled for input elements at the DOM level.  Valid only if the `msAbydos` flag is enabled. |
+| `msEdgeDesignerUI` | Use this flag to disable the Designer Shoreline App. This feature is not supported in WebView2. |
+| `msEdgeHubAppDesigner ` | Use this flag to disable the Designer Shoreline App. This feature is not supported in WebView2. |
+| `msEdgeDesignerDriverFix ` |  Use this flag to disable get content and auto show Designer feature. This feature is not supported in WebView2. |
```
Comment (mikehoffms) [inline microsoft-edge/webview2/concepts/webview-features-flags.md:110]:
```suggestion
| `msEdgeDesignerDriverFix ` |  Use this flag to disable getting content and automatically showing the Designer feature. This feature is not supported in WebView2. |
```
tech review this change @monica-ch

**Review by mikehoffms — APPROVED** [review]
Approved PR after Writer/Editor pass

**Review by nishitha-burman — APPROVED** [review]

**Pull request merged**